jiup/rules/rules.go

```diff
@@ -833,12 +833,11 @@ func init() {
 	Rule("kicad",
 		v.Regexp(
 			"http://kicad-pcb.org/download/windows/",
-			h.Re("Current Version: <strong>([0-9.]+)</strong>"),
```

Comment:
Why was this changed? On the website I see:
```
<p>Current Version: <strong>5.1.10</strong></p>
```

Reply:
Because Kicad seems to add _[0-9] to the exe files after the version number which the old regexp didnt handle